### PR TITLE
fix: updates web-tools for using NODE_ENV instead of args for static sites

### PR DESCRIPTION
**Pull request by mattlaguardia**
This PR changes our args process to use NODE_ENV key instead. This way in our vercel infra we can pass keys for different environments. 

Related to: https://github.com/zebrafishlabs/website_v2/pull/331

### CODE REVIEW

File: tools/pipelines/pipeline.html.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ module.exports = function setupHTMLPipeline(gulp) {
       return file.contents.toString();
 
     } else if (ext === '.js') {
-      if (args.env === 'production') {
+      if (process.env.NODE_ENV === 'production') {
```
Comment (luqven) [inline tools/pipelines/pipeline.html.js:40]: question(non-blocking): 

Does this not already pick up `process.env.NODE_ENV`? 

I think, ultimately, this change _is_ needed, but I remember testing this a while back and finding that, when no `--env` flag was present, it _did_ pick up the Node ENV vars. Is that not the case? Mostly just curious 🤔 .

**Review by luqven — APPROVED** [review]
LGTM 🚢 , I'm assuming you've tested locally.

If you'd like for me to clone and confirm, ping me, and I'll do that as well 👍🏼 .

File: tools/pipelines/pipeline.html.js (hunk 1 above)
Comment (mattlaguardia) [reply tools/pipelines/pipeline.html.js:40]: huh interesting I didnt see it picking it up on my local server, do you remember how you tested?

Comment (luqven) [reply tools/pipelines/pipeline.html.js:40]: Nah it's been far too long, but if you're not seeing it, then I must have been mistaken.

**Pull request merged**